src/api/parser/package.json

```diff
@@ -1,28 +1,65 @@
 {
-  "name": "parser",
+  "name": "@senecacdot/parser-service",
+  "private": true,
   "version": "1.0.0",
-  "description": "A service for parsing posts",
+  "description": "A service for parsing feeds",
   "scripts": {
-    "start": "node src/server.js"
+    "dev": "env-cmd -f env.local node src/index.js",
+    "start": "node src/index.js"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/Seneca-CDOT/telescope.git"
-  },
-  "author": "",
+  "repository": "Seneca-CDOT/telescope",
   "license": "BSD-2-Clause",
   "bugs": {
     "url": "https://github.com/Seneca-CDOT/telescope/issues"
   },
   "homepage": "https://github.com/Seneca-CDOT/telescope#readme",
   "dependencies": {
-    "@elastic/elasticsearch": "7.16.0",
-    "@elastic/elasticsearch-mock": "0.3.1",
-    "@senecacdot/satellite": "^1.x",
-    "bull": "3.29.3",
-    "bull-board": "2.1.3",
-    "jsdom": "18.1.1",
-    "normalize-url": "6.1.0",
-    "rss-parser": "3.12.0"
+    "@bull-board/api": "3.7.0",
+    "@bull-board/express": "3.7.0",
+    "@elastic/elasticsearch": "7.12.0",
+    "@elastic/elasticsearch-mock": "0.3.0",
+    "@senecacdot/satellite": "1.17.0",
+    "@wordpress/wordcount": "2.15.0",
```

Comment:
I don't think this is needed.  It was for the `/stats` endpoint, which we can remove.